lib/github/kv.rb

```diff
@@ -71,6 +71,7 @@ def initialize(encapsulated_errors = [SystemCallError], use_local_time: false, &
       @encapsulated_errors = encapsulated_errors
       @use_local_time = use_local_time
       @conn_block = conn_block
+      @table_name = GitHub::SQL::LITERAL(GitHub::DS.config.table_name).value
```

Comment:
Is there any benefit or safety trade off to saving the `table_name` as a `GitHub::SQL::LITERAL` verse interpolating it like other bind variables?

Reply:
Yeah I don't think it's necessary so I've removed it